exchanges/kraken/kraken_wrapper.go

```diff
@@ -328,60 +328,105 @@ func (k *Kraken) Run(ctx context.Context) {
 		}
 	}
 
-	if !k.GetEnabledFeatures().AutoPairUpdates && !forceUpdate {
-		return
+	if k.GetEnabledFeatures().AutoPairUpdates || forceUpdate {
+		if err := k.UpdateTradablePairs(ctx, forceUpdate); err != nil {
+			log.Errorf(log.ExchangeSys,
+				"%s failed to update tradable pairs. Err: %s",
+				k.Name,
+				err)
+		}
+	}
+
+	for _, a := range k.GetAssetTypes(true) {
+		if err := k.UpdateOrderExecutionLimits(ctx, a); err != nil && err != common.ErrNotYetImplemented {
+			log.Errorln(log.ExchangeSys, err.Error())
+		}
+	}
+}
+
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
+func (k *Kraken) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
+	if a != asset.Spot {
+		return common.ErrNotYetImplemented
```

Comment:
`common.ErrNotYetImplemented` 🤔Why?

Reply:
Cool. I was about to suggest asset.ErrNotSupported. But, this one makes sense. Thanks for reply @beadko